src/client.ts

```diff
@@ -1,3 +1,12 @@
+// =======================================================================
+//
+// DO NOT MODIFY THIS FILE BY HAND.
+//
+// Auto-generated by `@lune-climate/openapi-typescript-codegen`
+//
+// (You can modify this by hand if its extension is .hbs)
+//
+// =======================================================================
```

Comment:
question: did we actually not have this header before?

Reply:
No, it was included in this release of the base generator. We probably should have created a previous release for it but it was simply not the case.